.github/workflows/continuous-integration.yml

```diff
@@ -105,16 +66,22 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -r requirements.txt
+          # All dependencies but markdown-it-py
```

Comment:
Eventually this can be replaced with
```
pip install --upgrade "markdown-it-py${{ matrix.markdown }}" -e .[test]
```

Reply:
I see - but I also want to test without `markdown-it-py`, so I guess I will keep the current approach

Reply:
What do you mean by this? `markdown-it-py` is a hard requirement for this package right? If it's not, it should be moved to an extras.

Reply:
It is listed among the requirements, because we want the `myst-md` format to be available by default, but actually Jupytext can also work without `markdown-it-py` (and in that case the `Myst-md` format is not available).

Reply:
Well it's a design choice if you want to enforce a dependency or not. If you are enforcing it, keep up-to-date with upstream changes because you can cause dependency blockage for others to upgrade.